docs/modules/user-guide/partials/proc_devfile-registry.adoc

```diff
@@ -0,0 +1,50 @@
+[id="proc_devfile-registry_{context}"]
+= Devfile registry
+
+[role="_abstract"]
+A devfile registry is a service that stores and provides devfile stacks to Kubernetes developer tools like odo, Eclipse Che and the OpenShift Developer Console. Therefore, users can access devfiles through the devfile registry.  Devfile registries are based on the Oracle Cloud Infrastructure (OCI) Specification, and devfile stacks are stored as OCI-artifacts in the registry.
+
+Each devfile stack corresponds to a specific runtime or framework, such as,  NodeJS, Quarkus, and Wildfly. A devfile stack also contains resources, such as, a devfile.yaml, logo, and outerloop resources. These devfile stacks give users a template so they can get started developing with devfiles. For more information on the structure of a devfile registry, see https://github.com/johnmcollier/registry-docs/blob/main/registry-structure.adoc[registry-structure].
```

Comment:
As for the links, once we get all the devfile registry docs included in our docs repo, they can be xrefs (because they'll be linking to our own docs). For now, they'll have to be outside webpages that we're linking to.

Reply:
I was hoping you'd say that because that's my thought exactly! In the new commit, I'll remove the links, thanks!